code/modules/mob/mob_defines.dm

```diff
@@ -41,7 +41,7 @@
 	*/
 
 	/// The zone this mob is currently targeting
-	var/zone_selected = null
+	var/zone_selected = BODY_ZONE_CHEST
```

Comment:
I scanned through the uses of zone_selected to make sure nothing depended on it being null but I'm not 100% sure. Those usecases though probably shouldn't be doing that anyway. This is the best way to remove the need to update the mob selected zone in the update_icon proc of the ui.

Reply:
yeah, this was my concern, I'm pretty sure there is nothing at the mob level that will be fucked up by having a zone_selected